Cargo.toml

```diff
@@ -21,6 +21,7 @@ fstab = "0.4"
 log = "0.4"
 regex = "1.7"
 shellscript = "0.3"
+serde = { "version" = "1.0.145", features = ["derive"] }
```

Comment:
For this I generally like to keep the version to the minor instead of the patch. So `serde = { "version" = "1.0", features = ["derive"]}`